Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -107,11 +120,51 @@ class SyncNowSetting: WithAccountSetting {
         return attributedString
     }
 
+    private lazy var troubleshootButton: UIButton = {
+        let troubleshootButton = UIButton(type: UIButtonType.RoundedRect)
+        troubleshootButton.setTitle(Strings.FirefoxSyncTroubleshootTitle, forState: .Normal)
+        troubleshootButton.addTarget(self, action: #selector(self.troubleshoot), forControlEvents: .TouchUpInside)
+        troubleshootButton.tintColor = UIConstants.TableViewRowActionAccessoryColor
+        troubleshootButton.titleLabel?.font = DynamicFontHelper.defaultHelper.DefaultSmallFont
+        troubleshootButton.sizeToFit()
+        return troubleshootButton
+    }()
+
+    // TODO: This needs to be changed to the actual URL when we have it
+    private let syncSUMOURL = NSURL(string: "https://support.mozilla.org")!
```

Comment:
For the URL, use the `SupportUtils` class to generate a URL just like we do for What's New. You can ask Joni Savage to create a placeholder page on SUMO so that we already know the final URL.